bundler.go

```diff
@@ -150,8 +152,16 @@ func New(c *Configuration) (b *Bundler, err error) {
 	}
 
 	// Paths that depends on the input path
-	b.pathBuild = strings.TrimPrefix(strings.TrimPrefix(b.pathInput, filepath.Join(os.Getenv("GOPATH"), "src")), string(os.PathSeparator))
-	b.pathResources = filepath.Join(b.pathInput, "resources")
+	if strings.HasPrefix(b.pathInput, filepath.Join(os.Getenv("GOPATH"), "src")) {
+		b.pathBuild = strings.TrimPrefix(strings.TrimPrefix(b.pathInput, filepath.Join(os.Getenv("GOPATH"), "src")), string(os.PathSeparator))
+	}
+
+	if len(c.ResourcesPath) == 0 {
+		c.ResourcesPath = []string{"resouces"}
```

Comment:
And it should be `filepath.Join(b.pathInput, "resources")` not just `resources`

Reply:
see line 162.